API.md

```diff
@@ -211,13 +211,13 @@ Requests another confirmation email sent to the account's email address.
  - 401
  - 500
 
-### GET `/user/recover`
+### POST `/user/recover/request`
```

Comment:
not sure why this is a `POST` and not just a `GET` request with email as a query string param?

Reply:
It's a POST because it creates a record in the DB rather than reading one. Therefore according to the REST approach it needs to be a POST and not a GET.